addons/website_sale/static/src/js/website_sale.js

```diff
@@ -80,6 +80,9 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
                 $('.popover').on('mouseleave', function () {
                     self.$el.trigger('mouseleave');
                 });
+                self.cartQty = +$(data).find('.o_wsale_cart_quantity').text();
+                sessionStorage.setItem('website_sale_cart_quantity', self.cartQty);
```

Comment:
import from `@web/core/browser/browser` and use `sessionStorage` from that object instead of the one of the browser directly

Reply:
nope, actually browser.js isn't available in public bundles. too bad for testing